tools/gen-postmortem-metadata.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python
+
+#
+# gen-postmortem-metadata.py output_file.cc
+#
+# Creates debugging symbols to help naviage Node's internals using post-mortem
+# debugging tools.
+#
+
+import sys
+
+
+class DebugSymbol(object):
+    type_ = 'int'
+    _PREFIX = 'nodedbg_'
+
+    def __init__(self, name, value, headers=[], type_=None):
+        self.name = name
+        self.value = value
+        self.headers = headers
+        self.type_ = type_ or DebugSymbol.type_
```

Comment:
Why is this called `type_` instead of `type`?

Reply:
There's a built-in function named `type`, so using it as a name for variables or attributes is usually discouraged, but I could rename that if you prefer.